src/helm/benchmark/static/schema.yaml

```diff
@@ -2954,540 +2984,560 @@ run_groups:
+      what: text from Chinese social media 
+      who: web users
+      when: n/a
       language: Chinese
 
-  - name: cleva_subject_knowledge
-    display_name: CLEVA (Chinese) subject knowledge
-    description: CLEVA (Chinese) subject knowledge
+  - name: cleva_bias
+    display_name: CLEVA (Chinese) bias
+    description: Scenario that gauges bias of four demographic categories in dialogues, including race, gender, region, and occupation.
     metric_groups:
       - accuracy
       - calibration
+      - efficiency
       - general_information
+      - multiple_choice_classification_metrics
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: multiple-choice
+      what: short Chinese dialogues from social media
+      who: web users
+      when: n/a
       language: Chinese
 
-  - name: cleva_summarization
-    display_name: CLEVA (Chinese) summarization
-    description: CLEVA (Chinese) summarization
+  - name: cleva_copyright
+    display_name: CLEVA (Chinese) copyright
+    description: Scenario that measures copyright and memorization behavior for Chinese books and code, based off of [Carlini et al. (2021)](https://www.usenix.org/biblio-11958).
     metric_groups:
-      - accuracy
-      - calibration
-      - cleva_summarization_metrics
+      - copyright_metrics
       - general_information
+      - efficiency
     environment:
-      main_name: chinese_rouge_2
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: next-word prediction
+      what: books and code
+      who: n/a
+      when: n/a
       language: Chinese
 
-  - name: cleva_text_classification
-    display_name: CLEVA (Chinese) text classification
-    description: CLEVA (Chinese) text classification
+  - name: cleva_fact_checking
+    display_name: CLEVA (Chinese) fact checking
+    description: Scenario that lets models identify whether the given fact is true to test their factuality.
     metric_groups:
       - accuracy
       - calibration
+      - efficiency
       - general_information
+      - multiple_choice_classification_metrics
     environment:
       main_name: exact_match
       main_split: test
     taxonomy:
-      task: "?"
-      what: "?"
-      who: "?"
-      when: "?"
+      task: multiple-choice
+      what: factual statements in natural language form
+      who: n/a
+      when: n/a
       language: Chinese
 
-  - name: cleva_toxicity_detection
-    display_name: CLEVA (Chinese) toxicity detection
-    description: CLEVA (Chinese) toxicity detection
+# Others
+  - name: cleva_instruction_following
+    display_name: CLEVA (Chinese) instruction following
+    description: "Scenario that examines whether models could follow human instructions, mainly uncommon ones. It contains 'redefine' and 'pattern_matching_suppression' two subtasks."
```

Comment:
Change the second sentence to:
"It contains two subtasks: 'redefine' and 'pattern_matching_suppression'."